### ISSUE TITLE
Catalogue BTC

### ISSUE THREAD

**Issue by CelineAdl** [ochaillat/soaggyshopIssuesAndEnhancements#60]
Dans l'URLs BTC tu proposes 3 catalogues alors qu'ils ne doivent avoir accès que à celui: **particulier**

**Comment by rmvonji** [comment]
Oui, alors on pour le supprimer les autres dans le backend, je l'affiche juste en attendant que le catalogue particulier se remplisse
